### DISCUSSION TITLE
During the compilation process of kvrocks, the tools under the dependent rocksdb library are compiled together, which is convenient for operation and maintenance personnel to use rocksdb tools to solve problems (operation and maintenance personnel have little knowledge of c++).

### DISCUSSION THREAD

**Discussion by jcy1001** [apache/kvrocks#2136] [General]
During the compilation process of kvrocks, the tools under the dependent rocksdb library are compiled together, which is convenient for operation and maintenance personnel to use rocksdb tools to solve problems (operation and maintenance personnel have little knowledge of c++).

**Comment by PragmaTwice** [comment]
Sorry but if you want to use these rocksdb tools, I think you should build them from the rocksdb repo or find some trusted prebuilt binary.

These tools are not maintained by the kvrocks community.

**Comment by jihuayu** [comment]
You can use your package manager to install [Rocksdb-tools](https://pkgs.org/download/rocksdb-tools)
